backend/src/main/java/io/debezium/configserver/service/postgres/PostgresConnectorIntegrator.java

```diff
@@ -23,6 +29,96 @@
 // TODO: This will live in the PG connector module eventually
 public class PostgresConnectorIntegrator extends ConnectorIntegratorBase {
 
+    private static final Map<String, AdditionalPropertyMetadata> POSTGRES_PROPERTIES = new LinkedHashMap<>();
+    static {
+        // Connection properties
+        POSTGRES_PROPERTIES.put(PostgresConnectorConfig.SERVER_NAME.name(), new AdditionalPropertyMetadata(true, ConnectorProperty.Category.CONNECTION));
```

Comment:
As the map is static, let's make it immutable. I.e. add the contents to a temp var before, and then do `POSTGRES_PROPERTIES = Collections.unmodifiableMap(temp)`.

Reply:
thx, yes that's very good. Done!
I am available when you are @gunnarmorling